src/workerd/api/wpt/url-test.js

```diff
@@ -2,140 +2,136 @@
-    'response.formData() with input: †&†=x',
-    'request.formData() with input: %C2',
-    'response.formData() with input: %C2',
-    'request.formData() with input: %C2x',
-    'response.formData() with input: %C2x',
-    'request.formData() with input: _charset_=windows-1252&test=%C2x',
-    'response.formData() with input: _charset_=windows-1252&test=%C2x',
-    'request.formData() with input: ',
-    'response.formData() with input: ',
-    'request.formData() with input: a',
-    'response.formData() with input: a',
-    'request.formData() with input: a=b',
-    'response.formData() with input: a=b',
-    'request.formData() with input: a=',
-    'response.formData() with input: a=',
-    'request.formData() with input: =b',
-    'response.formData() with input: =b',
-    'request.formData() with input: &',
-    'response.formData() with input: &',
-    'request.formData() with input: &a',
-    'response.formData() with input: &a',
-    'request.formData() with input: a&',
-    'response.formData() with input: a&',
-    'request.formData() with input: a&a',
-    'response.formData() with input: a&a',
-    'request.formData() with input: a&b&c',
-    'response.formData() with input: a&b&c',
-    'request.formData() with input: a=b&c=d',
-    'response.formData() with input: a=b&c=d',
-    'request.formData() with input: a=b&c=d&',
-    'response.formData() with input: a=b&c=d&',
-    'request.formData() with input: &&&a=b&&&&c=d&',
-    'response.formData() with input: &&&a=b&&&&c=d&',
-    'request.formData() with input: a=a&a=b&a=c',
-    'response.formData() with input: a=a&a=b&a=c',
-    'request.formData() with input: a==a',
-    'response.formData() with input: a==a',
-    'request.formData() with input: a=a+b+c+d',
-    'response.formData() with input: a=a+b+c+d',
-    'request.formData() with input: %=a',
-    'response.formData() with input: %=a',
-    'request.formData() with input: %a=a',
-    'response.formData() with input: %a=a',
-    'request.formData() with input: %a_=a',
-    'response.formData() with input: %a_=a',
-    'request.formData() with input: %61=a',
-    'response.formData() with input: %61=a',
-    'request.formData() with input: %61+%4d%4D=',
-    'response.formData() with input: %61+%4d%4D=',
-    'request.formData() with input: id=0&value=%',
-    'response.formData() with input: id=0&value=%',
-    'request.formData() with input: b=%2sf%2a',
-    'response.formData() with input: b=%2sf%2a',
-    'request.formData() with input: b=%2%2af%2a',
-    'response.formData() with input: b=%2%2af%2a',
-    'request.formData() with input: b=%%2a',
-    'response.formData() with input: b=%%2a',
-  ],
-});
-export const urlSearchParamsAppend = run('urlsearchparams-append.any.js');
-export const urlSearchParamsConstructor = run(
-  'urlsearchparams-constructor.any.js',
-  {
+export default {
+  // TODO(soon): Implement globalThis.document
+  'a-element-origin.js': {
+    skipAllTests: true,
+  },
```

Comment:
I have a suggestion, let's change the type of this. If skipAllTests is true, let's require `comment: string` to exist, so we force the developer to write comment.

```suggestion
  'a-element-origin.js': {
    comment: 'Implement globalThis.document',
    skipAllTests: true,
  },
```

Reply:
Love it!